src/components/Display.js

```diff
@@ -1,28 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default class Display extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    const { input, next, operation } = this.props;
-    let number = '0';
-    if (operation && next === null) {
-      number = operation;
-    } else {
-      if (input) {
-        number = input;
-      }
-      if (next) {
-        number = next;
-      }
+const Display = ({ input, next, operation }) => {
+  let number = '0';
+  if (operation && next === null) {
+    number = operation;
+  } else {
+    if (input) {
+      number = input;
+    }
+    if (next) {
+      number = next;
     }
-    return <div className="input" id="display">{number}</div>;
   }
-}
+  return <div className="input" id="display">{number}</div>;
+};
+// export default class Display extends React.Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {};
+//   }
+
+//   render() {
+//     const { input, next, operation } = this.props;
+//     let number = '0';
+//     if (operation && next === null) {
+//       number = operation;
+//     } else {
+//       if (input) {
+//         number = input;
+//       }
+//       if (next) {
+//         number = next;
```

Comment:
- [ ] Kindly remove this commented code as this makes your codebase NOT clean.

Reply:
okay, thank you I will solve it soon.